module/src/test/java/com/iluwatar/module/FileLoggerModuleTest.java

```diff
@@ -0,0 +1,182 @@
+/**
+ * The MIT License Copyright (c) 2016 Amit Dixit
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and
+ * associated documentation files (the "Software"), to deal in the Software without restriction,
+ * including without limitation the rights to use, copy, modify, merge, publish, distribute,
+ * sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or
+ * substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT
+ * NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+ * DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+package com.iluwatar.module;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+import org.apache.log4j.Logger;
+import org.junit.Test;
+
+/**
+ * The Module pattern can be considered a Creational pattern and a Structural pattern. It manages
+ * the creation and organization of other elements, and groups them as the structural pattern does.
+ * An object that applies this pattern can provide the equivalent of a namespace, providing the
+ * initialization and finalization process of a static class or a class with static members with
+ * cleaner, more concise syntax and semantics.
+ * <p>
+ * The below example demonstrates a JUnit test for testing two different modules: File Logger and
+ * Console Logger
+ */
+public final class FileLoggerModuleTest {
+
+  private static final Logger LOGGER = Logger.getLogger(FileLoggerModuleTest.class);
+
+  private static final String OUTPUT_FILE = "output.txt";
+  private static final String ERROR_FILE = "error.txt";
+
+  private static final String MESSAGE = "MESSAGE";
+  private static final String ERROR = "ERROR";
+
+
+  /**
+   * This test verify that 'MESSAGE' is perfectly printed in output file
+   * 
+   * @throws IOException if program is not able to find log files (output.txt and error.txt)
+   */
+  @Test
+  public void positiveTestFileMessage() throws IOException {
```

Comment:
I would reconsider JUnit test method naming. See the other patterns for examples.